apps/examples/slsiwifi/Make.defs

```diff
@@ -1,37 +1,19 @@
-############################################################################
-# apps/examples/slsidemo/Make.defs
```

Comment:
Old license was wrong. We created the file but someone mistakenly copied the old nuttx header into the file instead of ours. Thats why I removed it. 
All files in these patches have been created by our team.